Formula/gping.rb

```diff
@@ -1,8 +1,8 @@
 class Gping < Formula
   desc "Ping, but with a graph"
   homepage "https://github.com/orf/gping"
-  url "https://github.com/orf/gping/archive/v1.2.1.tar.gz"
-  sha256 "ca11655f4357278a152476373cbc131b4183990e431196889c366706aedf6787"
+  url "https://github.com/orf/gping/archive/refs/tags/gping-v1.2.3.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/orf/gping/archive/gping-v1.2.3.tar.gz"
```

The vast majority of GitHub archive URLs in homebrew/core use `/archive/` instead of `/archive/refs/tags/`, so it's preferable to stick to the previous format. I'll have to create a brew PR to add this to the URL checks, as I'm surprised `brew style` doesn't already handle it.

Reply:
Fixed! Sorry 👍